firewood/src/merkle/stream.rs

```diff
@@ -465,6 +444,84 @@ mod tests {
         check_stream_is_done(merkle.iter(root)).await;
     }
 
+    #[tokio::test]
+    async fn no_start_key() {
```

Comment:
Is this somewhat overlapped with test at L407? If so, can we incorporate into one?

Reply:
Done